flexmeasures/ui/templates/crud/asset.html

```diff
@@ -147,6 +147,61 @@ <h3>Edit {{ asset.name }}</h3>
                 <span class="sr-only">Loading...</span>
             </div>
             <div id="sensorchart" class="card" style="width: 100%;"></div>
+            <div class="row">
+                <div class="copy-url" title="Click to copy URL to clipboard">
```

Comment:
More to the point: "Click to copy the URL to the current time range to clipboard."